pages/common/views/components/add-another.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component, Fragment } from 'react';
+
+class AddAnother extends Component {
+
+  constructor(props) {
+    super(props);
+    this.addAnother = this.addAnother.bind(this);
+  }
+
+  componentDidMount() {
+    const visible = Array(this.props.totalCount).fill(false);
+    visible.fill(true, 0, this.props.visibleCount);
+    this.setState({ visible });
+  }
+
+  addAnother(e) {
+    e.preventDefault();
+    const visible = [ ...this.state.visible ];
+    visible[visible.lastIndexOf(true) + 1] = true;
+    this.setState({ visible });
```

Comment:
you'll want to `return false` here if you don't want the `#` to end up in the URL

Reply:
Nah, `e.preventDefault()` will have caught that.